src/jsx/index.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path="../../node_modules/types-for-adobe/shared/PlugPlugExternalObject.ts" />
+import 'extendscript-es5-shim-ts'
```

Comment:
```suggestion
```

Let's put this in `tsconfig.json`

Reply:
The 'extendscript-es5-shim-ts' is not only type definitions, it also contains code (including JSON.stringify + JSON.parse), we have to import it in the code to also get the code loaded in

Reply:
There are no type definitions for JSON (since we are using `"noLib": true` now when bundling ExtendScript (for better correctness), the `extendscript-es5-shim-ts` package provides the types (and shim) for JSON and some others, I am not sure if we should still use the babel preset, I still have to look into that but at least they don't seem to conflict with eachother

Reply:
The babel preset is necessary if the user of the starter doesn't want to use Typescript.

I wonder if there should be 2 starter templates, one with Typescript ( and without babel ), and one with regular Javascript ( with babel and the preset )?

Also, Babel 7 can transpile Typescript now, so maybe we stick with babel and remove the typescript dependency?

I'll need to think more about it, but curious to know your thoughts.

Reply:
I'll keep this in for now. But I think it'd be good to eventually remove this requirement.